configs/efficientdet/retinanet_effd0_bifpn_1x_coco.py

```diff
@@ -0,0 +1,164 @@
+# model settings
+input_size = 512
+indices = (3, 4, 5)
+channels = 64
+num_levels = 5
+act_fn = 'silu'
+separable_conv = True
+num_epochs = 300
+delta = 0.1
+
+model = dict(
+    type='RetinaNet',
+    backbone=dict(
+        type='efficientnet_b0',
+        out_indices=indices,
+        frozen_stages=-1,
+        norm_eval=False,
+        pretrained=True,
+        verbose=False,
+    ),
+    neck=dict(
+        type='BiFPN',
+        in_channels=[40, 112, 320],
+        out_channels=channels,
+        input_indices=indices,
+        num_outs=num_levels,
+        num_layers=3,
+        weight_method='fast_attn',
+        act_fn=act_fn,
+        separable_conv=separable_conv,
+        epsilon=0.0001
+    ),
+    bbox_head=dict(
+        type='RetinaHead',
+        num_classes=90,
```

Comment:
Why 90?

Reply:
Number of classes used in original implementation ([link](https://github.com/google/automl/blob/262754b807610f947d57e0772757a8985b3b4895/efficientdet/hparams_config.py#L193))

Reply:
But actual number is 80, you just allocated more memory than needed. Have a look at other COCO-trained models in mmdetection.